package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "vets-json-schema",
-  "version": "25.1.2",
+  "version": "25.1.3",
```

Comment:
It's already at 25.1.3 in `master`: https://github.com/department-of-veterans-affairs/vets-json-schema/blob/master/package.json#L3